lib/data_update_scripts/20210211164634_fix_profile_field_edge_cases.rb

```diff
@@ -0,0 +1,9 @@
+module DataUpdateScripts
+  class FixProfileFieldEdgeCases
+    def run
+      ProfileField.where(attribute_name: %w[git_lab_url linked_in_url stack_overflow_url dribbble_url])
```

Comment:
should this also be `gitlab_url` here instead?

I also noticed that `stackoverflow_url` and `linkedin_url` are different in the `SPECIAL_SOCIAL_LINK_ATTRIBUTES`, is that supposed to be like that?

Reply:
I believe these values are actually correct. It always confuses me, but AFAIK this is how the data is actually stored. :upside_down_face: 

https://github.com/forem/forem/blob/ef481d07a33f91c2845ed8af4b51ab5cef217315/app/models/profile.rb#L38

Reply:
Yeah, these are correct, and that is actually what is prompting this PR. The whole thing is pretty unintuitive based on the whole `MAPPED_ATTRIBUTES` thing.